OTAnalytics/application/resources/resource_manager.py

```diff
@@ -195,6 +203,14 @@ class WorkspaceKeys(ResourceKey):
     SectionKeys.BUTTON_REMOVE: "Remove...",
     SectionKeys.TABLE_COLUMN_NAME: "Sections",
     SvzMetadataKeys.LABEL_SVZ_METADATA_FORM_HEADER: "SVZ",
+    SvzMetadataKeys.LABEL_TK_NUMBER: "Tk number",
```

Comment:
Use wording of CustomTkinter UI for all SVZ UI elements. The wording is taken from the official SVZ documentation.